tests/lib/rules/no-redundant-should-component-update.js

```diff
@@ -37,16 +37,6 @@ ruleTester.run('no-redundant-should-component-update', rule, {
       `,
       parserOptions,
     },
-    {
-      code: `
-        class Foo extends React.Component {
-          shouldComponentUpdate() {
-            return true;
-          }
-        }
-      `,
-      parserOptions,
-    },
```

Comment:
Two same in a row:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/no-redundant-should-component-update.js#L30-L49